pkg/runner/cluster_k8s.go

```diff
@@ -142,8 +146,22 @@ func (c *ClusterK8sRunner) Run(ctx context.Context, input *api.RunInput, ow *rpc
 
 	cfg := *input.RunnerConfig.(*ClusterK8sRunnerConfig)
 
+	// Right now we only support provider = "aws"; no provider defaults to "aws".
```

Comment:
We also support DockerHub, and we should get this to work on `kind` as well (i.e. cluster_k8s working without having to push an image, for example if you already specify an `artifact`)